frontend/lib/src/components/widgets/ChatInput/ChatInput.tsx

```diff
@@ -68,6 +72,7 @@ import {
 } from "@streamlit/lib/src/proto"
 import { set } from "lodash"
 import UploadedFiles from "../FileUploader/UploadedFiles"
+import { AttachFile } from "@emotion-icons/material-outlined"
```

Comment:
Not for now, but in the future, we would probably need to use icons from the Material Icon font rather than from the emotion pack. Let's keep it for now, and return to this later

Reply:
thanks for the callout, I'll check again later